Tutorial_Templates/tutorial_template_msb_v1.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<p>I wonder if we need to caveat that the Best Practices Guide is for NSIDC-internal folks only since it resides on our private confluence space. Maybe something like "Please read our Pedagogy Guide before creating your tutorial, as well as our Data Tutorials Best Practices Guide for those of you collaborating within NSIDC."</p><p> </p><p>And then for the next line, maybe that also needs to be clarified (since external contributors may not know who to go to within our DUE team, but instead we may want them to include that information in their pull request. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>